components/nimbus/android/build.gradle

```diff
@@ -3,9 +3,6 @@
 // READ THIS CAREFULLY before editing this file.
 // =============================================
 //
-// This is a lightly-modified version of the `build.gradle` file from Nimbus,
-// which you can find at `../external/nimbus-sdk/android/build.gradle`.
-//
 // The aim here is to pre-package the Nimbus Kotlin code and publish it under
 // `org.mozilla.appservices:nimbus` rather than `org.mozilla.experiments:nimbus`,
```

Comment:
nit: the comment here about using `appservices:nimbus` rather than `experiments:nimbus` and megazording can now be deleted, since there will no longer be anything published as `experiments:nimbus`.

Reply:
The whole comment block is essentially obsolete.  I've removed it.